agent/src/main/java/com/cloud/agent/Agent.java

```diff
@@ -432,9 +432,7 @@ protected void cancelTasks() {
             for (final WatchTask task : _watchList) {
                 task.cancel();
             }
-            if (logger.isDebugEnabled()) {
-                logger.debug("Clearing watch list: " + _watchList.size());
-            }
+            logger.debug("Clearing watch list: {}", _watchList.size());
```

Comment:
Either log the entire list, or change the log to specify that the size is being logged